__tests__/app.test.js

```diff
@@ -15,3 +23,33 @@ describe('GET /api', () => {
       });
   });
 });
+
+describe('GET /api/topics', () => {
+  test('200: Responds with an array of topic objects containing slug and description properties', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
```

Comment:
Is there another matcher we could use instead of `.toEqual(expect.objectContaining())`?

Reply:
Yes! Updated to `expect.toMatchObject` :raised_hands: